webapp/channels/webpack.config.js

```diff
@@ -141,6 +141,7 @@ var config = {
             crypto: require.resolve('crypto-browserify'),
             stream: require.resolve('stream-browserify'),
             buffer: require.resolve('buffer/'),
+            process: require.resolve('process/browser.js'),
```

Comment:
whats the reasoning with adding this?

Reply:
After upgrading the pdfjs-dist library webpack started throwing the following error:

```
[webapp] ERROR in ../node_modules/pdfjs-dist/legacy/build/pdf.mjs 4832:24-31
[webapp] Module not found: Error: Can't resolve 'process/browser' in '/Users/pabloandres/Documents/mm/mattermost/webapp/node_modules/pdfjs-dist/legacy/build'
[webapp] Did you mean 'browser.js'?
[webapp] BREAKING CHANGE: The request 'process/browser' failed to resolve only because it was resolved as fully specified
[webapp] (probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
[webapp] The extension in the request is mandatory for it to be fully specified.
```
This is due because after the pdfjs-dist 3 and up, it is no longer CJS, so to that module resolution failed because Webpack treats the file as a strict ESM which needs fully specified module requests including the extension.